pinot-core/src/test/java/org/apache/pinot/core/indexsegment/mutable/MutableSegmentImplTest.java

```diff
@@ -67,17 +69,20 @@ public void setUp()
     Assert.assertNotNull(resourceUrl);
     File avroFile = new File(resourceUrl.getFile());
 
-    SegmentGeneratorConfig config =
-        SegmentTestUtils.getSegmentGeneratorConfigWithoutTimeColumn(avroFile, TEMP_DIR, "testTable");
+    // Set the same default segment name for mutable and immutable segments
+    SegmentGeneratorConfig config = SegmentTestUtils
+        .getSegmentGeneratorConfigWithoutTimeColumn(avroFile, TEMP_DIR, MutableSegmentImplTestUtils.SEGMENT_NAME);
```

Comment:
Thy changing table name to segment name?

Reply:
After adding virtual columns to the mutable segment (which caught the [bug with +1](https://github.com/apache/incubator-pinot/pull/4774/files#r344978233)), the test complained the default segment name did not match between the mutable and immutable segment. So I added the hack to make them share the same segment name. Now I've change the test to allow the default segment name to be different.